kubernetes/charts/edge-kubernetes/values.yaml

```diff
@@ -83,6 +85,14 @@ edgeAgent:
     # literal "-" in liue of an empty string.
     # storageClassName: ""
 
+    # Set this to true if you wish to turn on experimental features support
+    # enableExperimentalFeatures: true
+    
+    # Set this to true if you wish to turn on Kubernetes create options support.
+    # In order to fully enable this experimental options for work you should
+    # set "true" both parameters "enableExperimentalFeatures" and "enableK8sExtensions"
```

Comment:
Suggestion:

Create options extensions for Kubernetes 
The feature allows the user to extend a module's create options with some Kubernetes primitives. To activate this experimental feature, both "enableExperimentalFeatures" and "enableK8sExtensions" must be set to "true".

Reply:
Done